blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -48,8 +48,9 @@ const SmallPromo = ({ customFields }) => {
     <>
       <article className="container-fluid small-promo">
         <div className="row">
-          {customFields.showHeadline && (
-            <div className={headlineClass}>
+          {customFields.showHeadline
+          && (customFields.headlinePosition === 'above' || undefined || 'undefined') && (
```

Comment:
```suggestion
          && (customFields.headlinePosition === 'above' || customFields.headlinePosition === undefined) && (
```

Reply:
this would be my guess, that 'undefined' string is evaluating as a truthy non-empty string. would kinda expect it to throw an error before then but not 100% sure

Reply:
I only removed the 'undefined' string in my suggestion because I'd only really expect `typeof headlinePosition === 'undefined'` to be plausible, rather than `headlinePosition === undefined`. Although I've been wrong about arc proptypes before